pootle/apps/pootle_store/models.py

```diff
@@ -2139,6 +2148,17 @@ def refresh_stats(self, include_children=True, cached_methods=None):
         languages and projects. For stores do nothing"""
         return
 
+    def all_pootle_paths(self):
+        """Get cache_key for all parents (to the Language and Project)
+        of current TreeItem
+        """
+        pootle_paths = super(Store, self).all_pootle_paths()
+
+        for vfcti in self.parent_vf_treeitems.iterator():
```

Comment:
There is the same the I mentioned in the comment above. Extra DB query if we don't use VF.